guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -31,15 +28,12 @@ ifdef::katello,satellite,orcharhino[]
 --activation-keys "_My_Activation_Key_"
 ----
 endif::[]
-. Log in to the host you want register and run the previously generated command.
-. Update subscription manager configuration for `rhsm.baseurl` and `server.hostname`:
+ifeval::["{context}" == "load-balancing"]
 +
-[options="nowrap" subs="+quotes,attributes"]
-----
-# subscription-manager config \
---rhsm.baseurl=https://loadbalancer.example.com/pulp/content \
---server.hostname=loadbalancer.example.com
-----
+Include the `--force` option to register a host that has been previously registered to a {SmartProxyServer}.
+. Replace the hostname in the URL of the generated command with the hostname and the {SmartProxy} port of the load balancer, such as `_loadbalancer.example.com_:{smartproxy_port}`.
```

Comment:
This is not needed, if you run `hammer host-registration generate-command  --smart-proxy-id ID` generated command is pointing to the load balancer:
```
curl -sS  'https://loadbalance.example.com:PORT/register'
```

Reply:
I get it know, this is needed only if you don't specify `--smart-proxy-id ID`